docs/source/examples/substra_core/titanic_example/run_titanic.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>I think we are not using matplolib on this specific example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>